ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -0,0 +1,106 @@
+<Toolbar>
+  <ToolbarActions>
+    {{! TODO remove action here in favor of cancel action in status banner }}
```

Comment:
this is pending design feedback, but I don't think it makes sense to hide/show this action based on the progress banner that will render on this page